client/actions/checks.js

```diff
@@ -20,12 +20,20 @@ function deleteCheck(status, payload = null) {
   }
 }
 
+function constructError(check, error) {
+  return {
+    ...error,
+    message: error.message,
+    checkId: check.checkId
+  }
+}
+
```

Comment:
Your call, we can refactor this later if we want to use custom errors more widely.